static/app-conf-projected.json

```diff
@@ -0,0 +1,99 @@
+{
+
+  "title": "Vue.js / OpenLayers WebGIS",
```

Comment:
What do you think of setting the title to
```suggestion
  "title": "Wegue - Projected Map",
```
so the title gives a clear hint what is shown?

Reply:
Ok.